assembl/graphql/schema.py

```diff
@@ -2003,11 +2003,11 @@ class Mutations(graphene.ObjectType):
 Schema = graphene.Schema(query=Query, mutation=Mutations)
 
 
-def print_schema_json(schema):
+def generate_schema_json():
```

Comment:
I imagine it was necessary for apollo-codegen? I don't really like the fact that it takes `Schema` from outside ;-)

Reply:
Not necessary. It just simplify the API. You can see the schéma.py module as a facade, and Schema as internal too.

Reply:
I added a generate_schema_json_from_schema(schema, output) api